### ISSUE TITLE
[BUG] Monobanks don't spawn

### ISSUE THREAD

**Issue by MaxWasUnavailable** [ChoiceTheorem/ChoiceTheorem-s-overhauled-village#179]
**Describe the bug**
Monobank structures seem to spawn, but they don't contain monobanks.

**To Reproduce**
Have this mod and Monobanks, and go through villages until you find one with a monobank structure.

**Expected behaviour**
Locked monobank blocks spawn, as they would without overhauled villages

**Minecraft Version**
1.20.1

**CTOV Version**
3.4.5

**Do you install Lithostitched?(1.20+)**
Yes

**Log**
Not applicable

**Additional context**
Not applicable

**Comment by MaxWasUnavailable** [comment]
Thanks!

**Comment by ChoiceTheorem** [comment]
I need more info here.
1. Does it monobank worked normally in vanilla villages.

**Comment by MaxWasUnavailable** [comment]
Yeah. Without this mod, monobanks spawn perfectly well in vanilla villages, both locked and unlocked variants, with loot inside of them.

**Comment by YnwLNE** [comment]
Can confirm, visited over 50 villages in multiple worlds and none of them had the monobanks, even when the config for the bank's weighting is extremely high. This also apparently affects wares as the same thing is happening

**Comment by ChoiceTheorem** [comment]
Unfortunately I can't work out what's wrong with it

**Comment by ChoiceTheorem** [comment]
This bug mysteriously disappear. I don't know how or why it does.
